main/zoe/api/zoe-contract-facet.md

```diff
@@ -2,25 +2,23 @@
 
 <Zoe-Version/>
 
-A Zoe Contract Facet is an API object for a running contract instance to access the Zoe state for that instance. A Zoe Contract Facet is accessed synchronously from within the contract, and usually is referred to in code as `zcf`. The contract instance is launched by `E(zoe).makeInstance`, and is given access to the `zcf` object during that launch. In the operation below, the `instanceHandle` is the handle for the running contract instance.
+A Zoe Contract Facet is an API object for a running contract instance. It accesses the instance's Zoe state. From within the contract, you synchronously access a Zoe Contract Facet (usually is referred to in code as `zcf`). The contract instance is launched and given access to the zcf object by `E(zoe).makeInstance`. In the operation below, the `instanceHandle` is the handle for the running contract instance.
+**tyg todo: There is no other instance of `instanceHandle` in this doc. Delete or rework?**
```

Comment:
Here's a simpler version: 

> Contracts interact with Zoe by sending messages to Zoe's Contract Facet (zcf). The contract can send messages zcf synchronously. The contract instance is launched and given access to the zcf object by a call to `E(zoe).makeInstance()`. 

The only external use of instanceHandle is for `getInstanceHandle()`. I don't know whether that's worth mentioning.

Reply:
Do you mean `zoe.getInstanceRecord(instanceHandle)`?